src/rgw/motr/gc/gc.cc

```diff
@@ -112,6 +160,21 @@ int MotrGC::dequeue(const DoutPrefixProvider* dpp, std::string iname, motr_gc_ob
   return rc;
 }
 
+int MotrGC::get_locked_gc_index(uint32_t* rand_ind)
+{
```

Comment:
I will suggest passing by reference instead of the pointer if we want to avoid the copy.
```suggestion
int MotrGC::get_locked_gc_index(uint32_t& rand_ind) {
```